src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -186,9 +186,13 @@ public abstract class FileWebRequestTestBase
             }
         }
 
+        protected virtual bool EnableConcurrentReadWriteTests => true;
+
         [Fact]
         public async Task ConcurrentReadWrite_ResponseBlocksThenGetsNullStream()
         {
+            if (!EnableConcurrentReadWriteTests) return;
```

Comment:
Sure, I could create an intermediate base abstract class for the async ones and put it there.